addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -505,7 +505,7 @@
             <div class="container">
                 <div class="row align-items-center">
                     <div class="col-lg-8 px-0">
-                        <img src="/mass_mailing_themes/static/src/img/theme_bold/s_default_image_image_text.jpg" class="img w-100" style="width: 533px;"/>
+                        <img src="/mass_mailing_themes/static/src/img/theme_bold/s_default_image_image_text.jpg" class="img w-100"/>
```

Comment:
Note for support team: This particular change is likely to be irrelevant for the customer, unless the issue that they are facing is specifically tied to this particular image in this particular template.